tools/api/README.md

```diff
@@ -5,15 +5,15 @@ First, create a `env` file with the following contents:
 ```
 export SERVER_URL=https://localhost:8080 # your fleet server url and port
 export CURL_FLAGS='-k -s' # set insecure flag
-export TOKEN=eyJhbGciOi... # your login token
+export TOKEN=eyJhbGciOi... # your api token
 ```
 
 Next set the `FLEET_ENV_PATH` to point to the `env` file. This will let the scripts in the `fleet/` folder source the env file.
 
 # Examples
 
 ```
-export FLEET_ENV_PATH=/Users/victor/fleet_env
+export FLEET_ENV_PATH=./fleet/code/fleet_env
```

Comment:
This looks perhaps backward (the other is `/code/fleet`. I'd suggest the same replacement, `/path/to/fleet/fleet_env`.

Reply:
Ahh yes, it should read `./code/fleet_env` without the extra `/fleet/` to match `./tools` command in this page that I was mimicking...

Should I update those as well? It's just super easy to run these scripts via copy and paste in the fleet directory if I don't use /path/to/fleet

Reply:
You could go that route, then I would just indicate before the example "From the Fleet directory:" then just list `/fleet_env`. Since `/code/fleet` is specific to your directory structure it wouldn't work if someone had Fleet installed somewhere else.